kurento-media-server/entrypoint.sh

```diff
@@ -73,3 +73,6 @@ cat /etc/hosts | sed '/::1/d' | tee /etc/hosts >/dev/null || true
 
 # Run Kurento Media Server
 exec /usr/bin/kurento-media-server "$@"
+
+# Run Avahi
+exec /etc/init.d/avahi-daemon start "$@"
```

Comment:
Doesn't the first `exec` replace the bach command by KMS, thus the second exec never gets executed?

Reply:
@j1elo, you are right, this line is not running and is not necessary, as Avahi runs as a service after installation.